orders/views.py

```diff
@@ -35,4 +38,14 @@ def admin_order_detail(request, order_id):
     order = get_object_or_404(Order, id=order_id)
     return render(request,
                   'admin/orders/order/detail.html',
-                  {'order':order})
\ No newline at end of file
+                  {'order':order})
+
+@staff_member_required
+def admin_order_pdf(request, order_id):
```

Comment:
```suggestion
def admin_order_pdf(request, order_id):
```

function name is not clear.